src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -45,10 +45,11 @@ export const selectActiveSwapperApiResponse: Selector<ReduxState, ApiQuote | und
     selectActiveSwapperName,
     (quotes, activeSwapperName) => {
       const selectedQuote = quotes.find(quote => quote.swapperName === activeSwapperName)
-      if (selectedQuote) {
+      if (selectedQuote?.quote !== undefined) {
```

Comment:
dont return the selected quote if it's in error, this fix prevents the UI showing error status when valid quotes are available